pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md

```diff
@@ -0,0 +1,50 @@
+| minio-operator          |                              |                                                           | No                  |
+| gatekeeper              |                              |                                                           | No                  |
+| kube-oidc-proxy         |                              |                                                           | Yes                 |
+| traefik                 |                              |                                                           |                     |
+| traefik-forward-auth    | cpu: 100m<br>memory: 128Mi   |                                                           | Yes                 |
+| velero                  |                              |                                                           | No                  |
+| kubetunnel              | 0.0.8                        |                                                           |                     |
+| logging-operator        | 3.15.0                       |                                                           |                     |
+| metallb                 | 0.12.2                       |                                                           |                     |
+| minio-operator          | 4.1.7                        |                                                           |                     |
+| nfs-server-provisioner  | 0.6.0                        |                                                           |                     |
+| nvidia                  | 0.4.3                        |                                                           |                     |
+| project-grafana-logging | 6.16.14                      |                                                           |                     |
+| project-grafana-loki    | 0.33.1                       |                                                           |                     |
+| project-logging         | 1.0.0                        |                                                           |                     |
+| prometheus-adapter      | 2.11.1                       |                                                           |                     |
+| reloader                | 0.0.99                       |                                                           |                     |
+| thanos                  | 0.4.5                        |                                                           |                     |
+| traefik                 | 10.3.0                       |                                                           |                     |
+| traefik-forward-auth    | 0.3.2                        |                                                           |                     |
+| velero                  | 3.1.1                        |                                                           |                     |
```

Comment:
There is a bunch of duplication here, plus the second column looks like version numbers instead of resource requirements.   Did we intend to have all this here?

Reply:
We still need to remove all the dups here...